nlp/app.py

```diff
@@ -0,0 +1,35 @@
+from flask import Flask, request, jsonify
+from sklearn.externals import joblib
+from pathlib import Path
+
+app = Flask(__name__)
+clf = None
+tf = None
+
+@app.before_first_request
+def load():
+    global clf
+    global tf
+    modelFile = Path.cwd() / 'nlp/tweets_model.pkl'
+    vectFile = Path.cwd() / 'nlp/tweets_vectorizer.pkl'
+
+    clf = joblib.load(modelFile)
+    tf = joblib.load(vectFile)
+
+@app.route('/', methods=['POST'])
+def main():
+    global clf
+    global tf
+
+    data = []
+    if request.method == 'POST':
+        message = request.form['message']
+        data.append(message)
+        prediction = clf.predict(tf.transform(data))
+        pred_proba = clf.predict_proba(tf.transform(data))
+
+        # TODO only return data if within certain confidence?
```

Comment:
i'll add this part next so it can return multiple emojis depending on confidence threshold